changelogs/fragments/allow_bad_things.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+    - Ebmed an overridable static santiization method into base inventory plugin class to allow individual plugins to override
```

Comment:
-> Embed an overridable static sanitization method into base inventory plugin class to allow individual plugins to override